### PR TITLE
Support :to_raise method for stub

### PR DESCRIPTION
**Pull request by s4ichi**
Support method `#to_raise`  like [webmock's](https://github.com/bblimke/webmock/blob/1a33665f254cb15dfd3702ff58728320b7f8930f/lib/webmock/request_stub.rb#L31-L36) one. It can use as following case:

```ruby
GrpcMock.stub_request("/hello.hello/Hello").to_raise(StandardError.new('message'))

client = Hello::Hello::Stub.new('localhost:8000', :this_channel_is_insecure)
client = client.hello(Hello::HelloRequest.new(msg: 'hi')) # => StandardError: message
```

### CODE REVIEW

File: lib/grpc_mock/response.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module GrpcMock
+  module Response
+    class InvalidArgument < StandardError; end
```
Comment (ganmacs) [inline lib/grpc_mock/response.rb]: why don't you use [ArgumentError](https://docs.ruby-lang.org/en/2.6.0/ArgumentError.html)? or is there any reason not to use it?

File: spec/grpc_mock/request_stub_spec.rb [hunk 2]
```diff
@@ -51,17 +57,56 @@
   describe '#to_return' do
     it 'registers response' do
       request = double(:request)
-      expect(GrpcMock::ResponsesSequence).to receive(:new).with([request]).once
+      expect(GrpcMock::ResponsesSequence).to receive(:new).with([anything]).once
```
Comment (ganmacs) [inline spec/grpc_mock/request_stub_spec.rb]: `with([GrpcMock::Response::Value])` seems bettter

File: spec/grpc_mock/request_stub_spec.rb [hunk 3]
```diff
@@ -51,17 +57,56 @@
...
+
+    context 'with invalid value (integer)' do
+      let(:response) { 1 }
+      it 'registers exception' do
+        expect { stub_request.to_raise(response) }.to raise_error(GrpcMock::Response::InvalidArgument)
+      end
+    end
+
+    it 'registers multi exception' do
+      exception = StandardError.new('message')
```
Comment (ganmacs) [inline spec/grpc_mock/request_stub_spec.rb]: Why do you not use `let` to define `exception` like other tests?

**Review by ganmacs — APPROVED** [review]
Yosasou

File: lib/grpc_mock/response.rb (hunk 1 above)
Comment (s4ichi) [reply lib/grpc_mock/response.rb]: I missed the ArgumentError. replaced it: https://github.com/ganmacs/grpc_mock/pull/5/commits/ba82f73a1b068c97673ad63a79cd9c2bf15357f1

File: spec/grpc_mock/request_stub_spec.rb (hunk 2 above)
Comment (s4ichi) [reply spec/grpc_mock/request_stub_spec.rb]: fixed: https://github.com/ganmacs/grpc_mock/pull/5/commits/e19d71175de5bd10c7bcd51fc581a1cbf8e75bab

File: spec/grpc_mock/request_stub_spec.rb (hunk 3 above)
Comment (s4ichi) [reply spec/grpc_mock/request_stub_spec.rb]: fix to use `let`: https://github.com/ganmacs/grpc_mock/pull/5/commits/e19d71175de5bd10c7bcd51fc581a1cbf8e75bab

**Comment by ganmacs** [comment]
released  0.3.1. Thank you.
https://github.com/ganmacs/grpc_mock/releases/tag/v0.3.1

**Pull request merged**